android/src/main/java/com/swmansion/reanimated/NativeProxy.java

```diff
@@ -113,6 +118,25 @@ public Scheduler getScheduler() {
     return mScheduler;
   }
 
+  private void toggleSlowAnimations() {
+  }
+
+  private void addDevMenuOption() {
+    ((ReactApplication) mContext.get().getApplicationContext())
+        .getReactNativeHost()
+        .getReactInstanceManager()
+        .getDevSupportManager()
+        .addCustomDevOption("Toggle slow animations - Reanimated", new DevOptionHandler() {
```

Comment:
looks more professional if you use `–` (ndash) instead of `-` or use parentheses:
```suggestion
        .addCustomDevOption("Toggle slow animations (Reanimated)", new DevOptionHandler() {
```

Reply:
BTW indeed, it seems like currently there's no option to dynamically change the text. The only available method for custom dev options is `addCustomDevOption`. I mean, we could do some dirty workaround, but let's keep it clean and nice.

Reply:
Fixed in [be69639](https://github.com/software-mansion/react-native-reanimated/pull/2931/commits/be696393eb60a1f2cdeb4c4664fdcc8e1bf055c5).